labs/RLSS19_TextWorld.ipynb

```diff
@@ -0,0 +1,2871 @@
+{
```

Comment:
<p>I don't have the same output (-&gt; "You have to open the gate first.", same as previous cell), did you run it afterwards?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/yfletberliac/rlss2019-hands-on/pull/1/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='18'/>

Reply:
<p>Same for me.</p>

Reply:
<p>Yep, that text is from my previous generated game, which turned out incompatible with the latest TextWorld. So, I switched from distributed game files to generated using fixed seed. I'll update this cell with upcoming commit. </p>